core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -168,7 +189,7 @@ class ClientQuotasRequestTest(cluster: ClusterInstance) {
   def testClientQuotasForScramUsers(): Unit = {
     val userName = "user"
 
-    val admin = cluster.admin()
+    val admin = createAdminClient()
```

Comment:
this `admin` is released in the final block, so I'm not sure if it is indeed the root cause :_

Also, this test case is disabled so does it cause the thread leak?